src/lighteval/utils/language.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import Enum
+
+
+class Language(Enum):
```

Comment:
How did you choose the language codes? In general it's better to use the ISO-639-2 standard (3 letter language codes, see [here](https://www.loc.gov/standards/iso639-2/php/code_list.php) for ex)

Reply:
Enums values are constants and should be all caps according to PEP8 (see examples in the [doc](https://docs.python.org/fr/3/library/enum.html)).
(So english -> ENGLISH, etc)